resources/dicts/patrols/general/training.json

```diff
@@ -9673,5 +9673,264 @@
+                ],
+		"stat_trait": ["compassionate", "thoughtful", "loving"],
+		"can_have_stat": ["p_l"],
+                "history_text": {
+                    "reg_death": "m_c was killed after training in the Dark Forest.",
+                    "scar": "m_c was scarred after a brutal training session in the Dark Forest."
+                },
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["platonic", "comfort", "respect"],
+                        "amount": 5
+                    }
+                ]
+            }
+        ],
+        "fail_outcomes": [
+            {
+                "text": "r_c says that no, no, {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't} want to fight <i>that</i>. The spirit is offended, but surprisingly, p_l sympathizes with the apprentice, and brings {PRONOUN/r_c/object} to do other safer training.",
+                "exp": 0,
+                "weight": 10,
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["platonic", "comfort", "trust"],
+                        "amount": 5
+                    }
+                ]
+            },
+            {
+                "text": "r_c completes the training session, but stumbles out of the forest and collapses on a field of grass.",
+                "exp": 0,
+                "weight": 10,
+                "dead_cats": ["r_c"],
+                "history_text": {
+                       "reg_death": "m_c was killed after training in the Dark Forest."
+                }
+            }
+        ]
+    },    
+    {
+        "patrol_id": "gen_train_solodarkforest",
+        "biome": ["Any"],
+        "season": ["Any"],
+        "types": ["training"],
+        "tags": [],
+        "patrol_art": "gen_train_darkforest2",
+        "min_cats": 1,
+        "max_cats": 1,
+        "min_max_status": {
+            "all apprentices": [-1, -1]
+        },
+        "pl_skill_constraint": ["DARK,2"],
+        "weight": 20,
+        "intro_text": "p_l stands at the entrance of a gloomy forest, draped in mud. A spirit waits, fur puffed up, claws unsheathed.",
+        "decline_text": "p_l turns tail and races away! {PRONOUN/p_l/poss} failure is jeered and laughed at by all the other Dark Forest spirits.",
+        "chance_of_success": 65,
+        "success_outcomes": [
+            {
+                "text": "p_l is met with a spirit who growls at {PRONOUN/p_l/object}. p_l pounces on them and wrestles them, and the two scuffle. p_l sinks {PRONOUN/p_l/poss} teeth in the Dark Forest spirit's neck, and they hiss, their body melting beneath p_l. Their pelt fades until there is no more, and p_l is alone, but not unhurt.",
```

Comment:
I like this, but i think permanently killing a dark forest cat should be a rare event (god I wish the code would let us make it happen too!). I'd rewrite this outcome to be that p_l fights them off and both retreat bleeding?

Reply:
And keep the original outcome as a very rare variant with like a weight of 2

Reply:
I'd also make a variant where p_l fights off the dark forest cat and the spirit is like good you have passed my test, but written so it's unclear whether it was actually a test or whether the dark forest cat just failed to assasinate p_l and is covering their own ass